src/verify/PhoneVerificationInputScreen.tsx

```diff
@@ -0,0 +1,135 @@
+import { StackScreenProps, useHeaderHeight } from '@react-navigation/stack'
```

Comment:
this file is mostly copy paste + remove some complexity from the VerificationInputScreen component